src/chi-vue/src/views/DataTableView.vue

```diff
@@ -40,6 +42,14 @@
           <ChiDataTableFilters :filtersData="toolbar.filtersData" class="-ml--2" />
         </ChiDataTableToolbar>
       </template>
+      <template #loadingSkeleton>
+        <div class="-d--flex -flex--column -w--100">
+          <div style="width: 420px;" class="chi-skeleton -mw--100"></div>
+          <div style="width: 960px;" class="chi-skeleton -xs -mw--100 -mt--2"></div>
+          <div style="width: 1024px;" class="chi-skeleton -xs -mw--100 -mt--1"></div>
+          <div style="width: 720px;" class="chi-skeleton -xs -mw--100 -mt--1"></div>
```

Comment:
Question: aren't there any utility classes which can replace these inline styles? Example: -w--960. If not..does it make sense to include them using some predefined sizes like -mw does?

Reply:
We do have max-width sizing utility classes like `-mw--480` but not width ones like `-w--480`.
@jllr Thoughts on the suggestion of adding this kind of adding width utility classes with predefined sizes?

Reply:
Replaced the inline styles with -w--* utility classes and made them responsive to screen size. Thanks for the suggestion!